### PR TITLE
Getting Started notebook and docs changes

### PR DESCRIPTION
**Pull request by okuchaiev**
Signed-off-by: Oleksii Kuchaiev <okuchaiev@nvidia.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,6 +47,44 @@ Requirements
 
 1) Python 3.6, 3.7 or 3.8
 2) Pytorch 1.7.1.  WARNING: This version currently does not support Pytorch 1.8.0
```
Comment (titu1994) [inline README.rst:49]: Hmm maybe we need this warning in a seperate line? Just a suggestion.

File: docs/source/starthere/intro.rst [hunk 2]
```diff
@@ -30,6 +30,54 @@ Requirements
...
+    # After this you should see English translation
+    # Let's convert it into audio
+    # A helper function which combines Tacotron2 and WaveGlow to go directly from
+    # text to audio
+    def text_to_audio(text):
+      parsed = spectrogram_generator.parse(text)
+      spectrogram = spectrogram_generator.generate_spectrogram(tokens=parsed)
+      audio = vocoder.convert_spectrogram_to_audio(spec=spectrogram)
+      return audio.to('cpu').numpy()
+    audio = text_to_audio(english_text[0])
```
Comment (titu1994) [inline docs/source/starthere/intro.rst:80]: no way to hear audio in this step - it just returns a tensor - maybe point to librosa to write it to a file?

File: tutorials/NeMo_Getting_Started.ipynb [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+   "execution_count": null,
+   "metadata": {
+    "id": "1h9nhICjA5Dk"
+   },
+   "outputs": [],
+   "source": [
+    "# Speech Recognition model - QuartzNet trained on Russian part of MCV 6.0\n",
+    "quartznet = nemo_asr.models.EncDecCTCModel.from_pretrained(model_name=\"stt_ru_quartznet15x5\").cuda()\n",
+    "# Neural Machine Translation model\n",
+    "nmt_model = nemo_nlp.models.MTEncDecModel.from_pretrained(model_name='nmt_ru_en_transformer6x6', strict=False).cuda()\n",
```
Comment (titu1994) [inline tutorials/NeMo_Getting_Started.ipynb:113]: Is the strict=False necessary?

**Review by titu1994 — APPROVED** [review]
Overall looks good, minor comments that need could be addressed.

File: docs/source/starthere/intro.rst (hunk 2 above)
Comment (okuchaiev) [reply docs/source/starthere/intro.rst:80]: >>" maybe point to librosa to write it to a file?"
- Do you have a code snippet to do this? (It uses IPython in the notebook)

File: tutorials/NeMo_Getting_Started.ipynb (hunk 3 above)
Comment (okuchaiev) [reply tutorials/NeMo_Getting_Started.ipynb:113]: yes, I will undo it once NMT models are re-published.

File: docs/source/starthere/intro.rst (hunk 2 above)
Comment (titu1994) [reply docs/source/starthere/intro.rst:80]: Ah that's a problem. I think for now it's fine to put an ipython snippet but I'll try finding a way to do it in librosa

**Pull request merged**